Cohort2/team4/server/tsconfig.json

```diff
@@ -3,8 +3,7 @@
     "outDir": "dist",
     "sourceMap": true,
     "target": "esnext",
-    "module": "esnext",
-    "moduleResolution": "node",
+    "module": "CommonJS",
```

Comment:
So we want to switch to using ESM. This is fine for now, but I will make the change to main after this merges to get us away from CommonJS.